app.js

```diff
@@ -0,0 +1,134 @@
+// Book Class: Represents a Book
+class Book {
+  constructor(title, author, page) {
+    this.title = title;
+    this.author = author;
+    this.page = page;
+  }
+}
+
+// UI Class: Handle UI Tasks
+class UI {
+  static displayBooks() {
+    const books = Store.getBooks();
+
+    books.forEach((book) => UI.addBookToList(book));
+  }
+
+  static addBookToList(book) {
+    const list = document.querySelector('#book-list');
+
+    const row = document.createElement('tr');
+
+    row.innerHTML = `
+      <td>${book.title}</td>
+      <td>${book.author}</td>
+      <td>${book.page}</td>
+      <td><a href="#" class="btn btn-danger btn-sm delete">X</a></td>
+    `;
+
+    list.appendChild(row);
+  }
+
+  static deleteBook(el) {
+    if (el.classList.contains('delete')) {
+      el.parentElement.parentElement.remove();
+    }
+  }
+
+  static showAlert(message, className) {
+    const div = document.createElement('div');
+    div.className = `alert alert-${className}`;
```

Comment:
Let's separate the DOM from application logic, you can create another file for DOM

![image](https://user-images.githubusercontent.com/31897434/87004592-9027f300-c1b5-11ea-9e2d-6617ca2c92b2.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59623451/